### DISCUSSION TITLE
long-running-process in example does not work

### DISCUSSION THREAD

**Discussion by Weicai-Ling** [cockpit-project/cockpit#20473] [Q&A]
hello experts: 

I copied the package of long-running-process from (https://github.com/cockpit-project/cockpit/tree/main/examples/long-running-process) to `/usr/share/cockpit/long` , the "Start" buttern is grey on Cockpit web page. It shows the same error when I copy the package to my user directory: `/home/usr/.local/share/cockpit/long ` . I am running cockpit 300.1 installed with software selection of "Administration Graphic Tools" in beginning of Rocky Linux 9.3 OS installation. 

Who could please advise how I can solve this problem ? 

Thanks, 
Weicai-Ling

**Comment by jelly** [comment]
Tried this and it works, what error do you get? Is it a JavaScript error in the browser developer tools?

![image](https://github.com/cockpit-project/cockpit/assets/67428/d0674bd0-cd2d-487f-8b8f-729daf8858f7)

**Comment by martinpitt** [comment]
We [test this page on all OSes](https://github.com/cockpit-project/cockpit/blob/main/test/verify/check-examples#L84), so indeed we need a lot more details -- it's not obvious what is/could be broken.
